### PR TITLE
SWARM-589 introduce simple Arquillian CommandService impl.

### PR DESCRIPTION
**Pull request by tremes**

### CODE REVIEW

**Comment by kenfinnigan** [comment]
test this please

**Comment by kenfinnigan** [comment]
ok to test

File: arquillian/api/src/main/java/org/wildfly/swarm/arquillian/adapter/WildFlySwarmCommandService.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+package org.wildfly.swarm.arquillian.adapter;
+
+import org.jboss.arquillian.container.test.spi.command.Command;
+import org.jboss.arquillian.container.test.spi.command.CommandService;
+
+public class WildFlySwarmCommandService implements CommandService {
+    @Override
+    public <T> T execute(Command<T> command) {
+        if (command.getThrowable() != null) {
+            new RuntimeException(command.getThrowable());
```
Comment (gastaldi) [inline arquillian/api/src/main/java/org/wildfly/swarm/arquillian/adapter/WildFlySwarmCommandService.java]: Looks like this line is missing a `throw` keyword?

Comment (tremes) [inline arquillian/api/src/main/java/org/wildfly/swarm/arquillian/adapter/WildFlySwarmCommandService.java]: Yes.:-) Thank you.

**Pull request merged**